### PR TITLE
Feature/ci 125 add my connections

### PR DESCRIPTION
**Pull request by JakubJankowski**
- Link to jira's task: https://nomtek.atlassian.net/browse/CI-125
- APIClient should now fetch My Connections

### CODE REVIEW

File: CreatubblesAPIClientTests/Spec/User/MyConnectionRequestSpec.swift [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                            expect(users).notTo(beNil())
+                            expect(pageInfo).notTo(beNil())
+                            done()
+                        })
+                    }
+                }
+            }
+            
+            //Should it return an error?
+            it("Should return error when not logged in")
```
Comment (JakubJankowski) [inline CreatubblesAPIClientTests/Spec/User/MyConnectionRequestSpec.swift:42]: Should an error be returned here? At this point we don't receive an error here, so this test fails.

File: CreatubblesAPIClientTests/Spec/Content/ContentResponseHandlerSpec.swift [hunk 2]
```diff
@@ -65,6 +65,32 @@ class ContentResponseHandlerSpec: QuickSpec
                     }
                 }
             }
+            
+            it("Should return Connected contents after login")
+            {
```
Comment (JakubJankowski) [inline CreatubblesAPIClientTests/Spec/Content/ContentResponseHandlerSpec.swift:70]: Even though this test passes, returned entries are always an empty array. Is that working as intended?

File: CreatubblesAPIClientTests/Spec/User/MyConnectionRequestSpec.swift (hunk 1 above)
Comment (mmiedlarz) [inline CreatubblesAPIClientTests/Spec/User/MyConnectionRequestSpec.swift:42]: If i run only this one test case, everything works fine (error is returned here), but when I run all the tests, it fails. I guess that tests can be run simultaneously which can cause this issue.

File: CreatubblesAPIClientTests/Spec/Content/ContentResponseHandlerSpec.swift (hunk 2 above)
Comment (mmiedlarz) [inline CreatubblesAPIClientTests/Spec/Content/ContentResponseHandlerSpec.swift:70]: It depends on the account. I checked it with another one (PM me for details), and everyghing was ok

**Comment by mmiedlarz** [comment]
Code 👍 
Func 👍  (we should think how we can fix issue with simultaneous tests)

**Comment by mmiedlarz** [comment]
I realized, that you should add public interface before merge.

**Pull request merged**